compile-run-with-classpath/solution.sh

```diff
@@ -7,3 +7,6 @@
 # First argument is blank: true
 # Second argument is blank: true
 # Third argument is blank: false
+
+javac -classpath commons-lang3-3.9.jar StringIsBlank.java
+java -cp commons-lang3-3.9.jar:. StringIsBlank  "" \t '*'
```

Comment:
```suggestion
java -cp commons-lang3-3.9.jar:. StringIsBlank  "" '' '*'
```

`\t`严格意义上不是一个空格，只能说它是一个“空白字符”